report/report.py

```diff
@@ -547,4 +518,12 @@ def handler(event=None, context=None) -> dict:
 
 if __name__ == "__main__":
 
-    print(handler())
+    load_dotenv()
+
+    connection = get_db_connection()
+
+    yesterday_data = load_yesterday_data(connection)
+
+    html_string = generate_html_string(yesterday_data)
+
+    convert_html_to_pdf(html_string, './Bandcamp-Daily-Report.pdf')
```

Comment:
Very nice separation of code into functions! Makes sections like this look really nice